index.js

```diff
@@ -143,6 +144,17 @@ instance.prototype.actions = function (system) {
 				regex: self.REGEX_NUMBER
 			}*/]
 		},
+		'volume_num': {
+			label: 'Volume(specified)',
+			options: [ {
+				type: 'textinput',
+				label: 'Volume level 00-99 (80 = 0db, 00 = -80db, 99 = mute)',

```

Comment:
I'm not sure what to do here, as I expect this is to spec ... Its rather confusing that the highest possible value is mute.  I almost want to suggest a dropdown with each dB (just skip the decimal value capability).  Also, I'd recommend using regex to enforce 0-99 instead of simply a number if this is left as a textinput.

Reply:
Ha, it's even more confusing. you can do half db values with 3 digits. So 795 is -0.5dB.

Reply:
Ugh ... let's do a dropdown.  You need help with that at all?  I'd recommend using a loop to push the values and push mute manually after.

Reply:
Yes, i could definitely use some help with that. Also struggling with how to add regex's. Wanted to add a [0-9][0-9] limit, but it's not taking.

Reply:
Try this `regex: '/[0-9][0-9]/ `